### PR TITLE
Child safety module

### PR DESCRIPTION
**Pull request by devcolonel**

### CODE REVIEW

File: opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java [hunk 1]
```diff
@@ -383,4 +385,30 @@ protected void evaluateExclusiveBreastFeeding(Map<String, ServiceWrapper> servic
                 .build();
         actionList.put(title, action);
     }
+
+    private void evaluateChildSafety(Map<String, ServiceWrapper> serviceWrapperMap) throws Exception {
+        ServiceWrapper serviceWrapper = serviceWrapperMap.get("Child Safety");
```
Comment (maulidkategile) [inline opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java:458]: The referenced recurring service contains a lowercase letter for in the keyword "safety", resulting in a null serviceWrapper object

**Review by maulidkategile — CHANGES_REQUESTED** [review]

**Review by maulidkategile — APPROVED** [review]

**Comment by issyzac** [comment]
@devcolonel please work on the conflicts so that the changes can be merged

File: opensrp-chw/src/main/java/org/smartregister/chw/actionhelper/ChildHVChildSafetyActionHelper.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    @Override
+    public String evaluateSubTitle() {
+        child_safety_counselled = getYesNoTranslation(child_safety_counselled);
+        return MessageFormat.format("{0}: {1}", context.getString(R.string.child_safety), child_safety_counselled);
+    }
+
+    public String getYesNoTranslation(String subtitleText) {
+        if ("yes".equals(subtitleText)) {
+            return context.getString(R.string.yes);
+        } else if ("no".equals(subtitleText)) {
```
Comment (issyzac) [inline opensrp-chw/src/main/java/org/smartregister/chw/actionhelper/ChildHVChildSafetyActionHelper.java]: In the json file the key is different from what you are equating it to here

**Review by issyzac — CHANGES_REQUESTED** [review]
Can you run and test and make sure the service is available

**Review by issyzac — APPROVED** [review]
Looks good.

**Pull request merged**